### PR TITLE
Fix 2 broken links in the docs

### PR DESCRIPTION
**Pull request by qlijin**
There is 2 link is broken(404) in the docs.

> https://github.com/kubernetes/community/blob/master/communication.md#social-media
> https://github.com/kubernetes-sigs/etcdadm/etcd-manager

update to:

> https://github.com/kubernetes/community/tree/master/communication#slack
> https://github.com/kubernetes-sigs/etcdadm/tree/master/etcd-manager

### CODE REVIEW

File: docs/contributing/index.md
Comment (kundan2707) [inline docs/contributing/index.md:62]: This link should be kubernetes of kops-dev channel

File: docs/operations/etcd_backup_restore_encryption.md
Comment (kundan2707) [inline docs/operations/etcd_backup_restore_encryption.md:99]: Thanks for providing correct link

**Review by rifelpet — APPROVED** [review]
thanks!

**Comment by rifelpet** [comment]
@qlijin you'll need to squash your commits before this PR can be merged

/hold

**Comment by qlijin** [comment]
The squash operation is completed.

**Review by rifelpet — APPROVED** [review]
/hold cancel

**Pull request merged**
